api/envoy/api/v2/route/route.proto

```diff
@@ -1139,7 +1139,7 @@ message RateLimit {
     //
     // .. code-block:: cpp
     //
-    //   ("<descriptor_key>", "<header_value_queried_from_header>")
+    //   ("request_headers", "<header_value_queried_from_header>")
```

Comment:
I don't think this is correct. The configured descriptor key is used from my look at the code. Can you describe why you think this is wrong?

Reply:
As the envoy Ratelimit doc mentiond [here](https://www.envoyproxy.io/docs/envoy/latest/api-v2/api/v2/route/route.proto#route-ratelimit-action-requestheaders) , example action configuration for request headers is 

`("<descriptor_key>", "<header_value_queried_from_header>")`

, while the other actions are 

`("source_cluster", "<local service cluster>")`
`("destination_cluster", "<routed target cluster>")`
`("remote_address", "<trusted address from x-forwarded-for>")`
`("generic_key", "<descriptor_value>")`
`("header_match", "<descriptor_value>")`

I was confused about the `<descriptor_key>` , which i thinik should be `request_headers`,
so the example request header action should be like below:
`("request_headers", "<header_value_queried_from_header>")`

Reply:
I don't think that is true. `<descriptor_key>` is filled with the `descriptor_key` field below. Please feel free to update the docs to be more clear on this.

/wait